src/plugins/tjdashboardrenderer/countbox/countbox.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension version="3.1" type="plugin" group="tjdashboardrenderer" method="upgrade">
+	<name>PLG_TJDASHBOARDRENDERER_COUNTBOX</name>
+	<author>Techjoomla</author>
+	<creationDate>13th September 2017</creationDate>
+	<copyright>(C)techjoomla.com</copyright>
+	<license>http://www.gnu.org/copyleft/gpl.html GNU/GPL</license>
```

Comment:
@manojLondhe can you check the xml meta and it's value

Reply:
@monikaatvowel use `<license>http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL</license>`

Reply:
@manojLondhe ok sir